docs/credits.md

```diff
@@ -1415,7 +1416,7 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | Octopus.Server.MessageContracts                   | Octopus Deploy                                                            | [https://github.com/OctopusDeploy/Server.MessageContracts](https://github.com/OctopusDeploy/Server.MessageContracts)                                                                                                        | [Apache-2.0](https://licenses.nuget.org/Apache-2.0)                                                                 |
 | Octopus.Server.Portal                             | @                                                                         | []()                                                                                                                                                                                                                        | [Undefined]()                                                                                                       |
 | Octopus.Shared                                    | Octopus Deploy                                                            | [https://github.com/OctopusDeploy/OctopusShared](https://github.com/OctopusDeploy/OctopusShared)                                                                                                                            | [Apache-2.0](https://licenses.nuget.org/Apache-2.0)                                                                 |
-| Octopus.Tentacle.CrossPlatformBundle              | agentuser@win19sql19-a4                                                   | []()                                                                                                                                                                                                                        | [Undefined]()                                                                                                       |
+| Octopus.Tentacle.CrossPlatformBundle              | agentuser@EC2AMAZ-5SK3MFQ                                                 | []()                                                                                                                                                                                                                        | [Undefined]()                                                                                                       |
```

Comment:
I've [raised this](https://octopusdeploy.slack.com/archives/CNHBHV2BX/p1637896022477100) with F&M